core/services/pipeline/task.cborparse.go

```diff
@@ -0,0 +1,48 @@
+package pipeline
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/core/store/models"
+)
+
+type CBORParseTask struct {
```

Comment:
sorry what uses this again? Is it that some people stuff json objects into a solidity ```bytes``` parameter?

Reply:
Yeah, in direct request jobs the `bytes data` field is a CBOR-encoded map